utils/test/src/main/kotlin/Matchers.kt

```diff
@@ -119,3 +125,27 @@ fun matchExpectedResult(
         )
     }
 }
+
+infix fun String?.shouldMatchJsonSchema(schemaJson: String) = this should matchJsonSchema(schemaJson)
```

Comment:
I'd prefer to drop this and consistently use `should matchJsonSchema(schemaJson)` only.